content/en/faqitems/eks-1-how-to-do-fault-injection.markdown

```diff
@@ -0,0 +1,8 @@
++++
+question = "How do I perform fault injection with Bottlerocket?"
+group = "EKS"
```

Comment:
Some possibilities, what do you think?

- Kubernetes on Bottlerocket
- Cluster Testing
- Stress Testing
- Workload Testing
- Workloads
- Auditing and Validation

I'm leaning towards "Kubernetes on Bottlerocket" (since Istio/Chaos Mesh/AWS FIS are Kubernetes fault injection tools)

Reply:
Maybe just 'Kubernetes'? Bottlerocket is implied.